app/src/main/res/values/styles.xml

```diff
@@ -2,75 +2,197 @@
+        <item name="android:layout_marginBottom">@dimen/basic_marge</item>
+        <item name="android:textColor">@color/custom_grey</item>
+        <item name="android:textAppearance">@style/TextAppearance.RealEstateManagerStyles.Body1
+        </item>
     </style>
 
-    <style name="Widget.customViewAddPropertySoldEditTextStyle" parent="Widget.AppCompat.EditText">
-        <item name="android:layout_marginEnd">@dimen/big_marge</item>
-        <item name="android:inputType">date</item>
+    <style name="Widget.FragmentAddPropertyDescriptionTitleTextView" parent="Widget.Headline1TextView">
+        <item name="android:layout_marginStart">@dimen/basic_marge</item>
+        <item name="android:text">@string/description</item>
     </style>
 
-    <style name="Widget.customViewDescriptionTextStyle" parent="Widget.MaterialComponents.TextView">
-        <item name="android:maxLines">8</item>
-        <item name="android:text">@string/description_text</item>
-        <item name="android:textAppearance">@style/TextAppearance.RealEstateManagerStyles.Body1</item>
+    <!-- Recycler view -->
+    <style name="Widget.RecyclerViewPropertyListItemPropertyTypeTextView" parent="Widget.Headline1TextView">
+        <item name="android:text">@string/property_type</item>
+    </style>
+
+    <style name="Widget.RecyclerViewPropertyListItemTownTextTextView" parent="Widget.Subtitle1TextView">
+        <item name="android:text">@string/town</item>
+    </style>
+
+    <style name="Widget.RecyclerViewPropertyListItemPropertyPriceTextView" parent="Widget.Headline1TextView">
+        <item name="android:text">@string/price</item>
+        <item name="android:textColor">@color/custom_pink</item>
+        <item name="android:layout_marginStart">0dp</item>
+        <item name="android:paddingBottom">0dp</item>
     </style>
     <!--__________________________________RECYCLER VIEW__________________________________-->
 
-    <style name="Widget.listRecyclerviewPhotoDescriptionTextViewStyle" parent="Widget.MaterialComponents.TextView">
+    <style name="Widget.listRecyclerviewPhotoDescriptionTextView" parent="Widget.MaterialComponents.TextView">
         <item name="android:text">@string/description</item>
-        <item name="android:textAppearance">@style/TextAppearance.RealEstateManagerStyles.Body1</item>
+        <item name="android:textAppearance">@style/TextAppearance.RealEstateManagerStyles.Body1
+        </item>
         <item name="android:paddingBottom">15dp</item>
         <item name="android:paddingTop">15dp</item>
         <item name="android:paddingEnd">0dp</item>
-        <item name="android:paddingStart">15dp</item>
         <item name="android:textColor">@color/white</item>
         <item name="android:background">@color/black</item>
-        <item name="android:alpha">0.5</item>
+        <item name="android:alpha">0.6</item>
+    </style>
+
+    <!--__________________________________TOP BAR__________________________________-->
+
```

Comment:
Use MaterialComponentsButton instead of AppCompat.

Reply:
done